src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -2025,7 +2037,7 @@ public boolean scheduleBuild(int quietPeriod, Cause c, Action... actions) {
      * {@inheritDoc}
      */
     public void checkAbortPermission() {
-        checkPermission(AbstractProject.ABORT);
+        checkPermission(Item.CANCEL);
```

Comment:
Separate to this PR ... seems like `ABORT` is deprecated, so shouldn't these methods be deprecated (and renamed perhaps)?